gdcdictionary/schemas/follow_up.yaml

```diff
@@ -1693,18 +1836,33 @@ properties:
       - Seizure
       - Sensory Changes
       - Serous tubal intraepithelial carcinoma (STIC)
+      - Sialadenitis
+      - Squamous Metaplasia
       - Steatosis
       - Tattoo
       - Thyroid Nodular Hyperplasia
       - Tobacco, NOS
       - Tobacco, Smokeless
       - Tobacco, Smoking
+      - Tubulointerstitial Disease
+      - Tumor-associated lymphoid proliferation
```

Comment:
Should "associated", "lymphoid", "proliferation" be capitalized? @tlicht3 @ganjigajanan